### ISSUE TITLE
travis CI? 

### ISSUE THREAD

**Issue by export-mike** [fmoo/react-typeahead#202]
thoughts? its free for public repos

**Comment by fmoo** [comment]
Yeah, this would be great to have.   I've only set up travis once (for a non-JS project) and haven't had the cycles to figure it for this yet.

**Comment by export-mike** [comment]
sure I'll add it to my todo, it'll be a good way for me to understand the build process
